consensus/msgs_test.go

```diff
@@ -314,7 +314,7 @@ func TestWALMsgProto(t *testing.T) {
 	}
 }
 
-// nolint:lll //ignore line length for tests
+//nolint:lll //ignore line length for tests
```

Comment:
```suggestion
//nolint:lll //ignore line length for tests
```

Pretty sure this isn't needed here.